### PR TITLE
atan2, hypo and more label

### PR DESCRIPTION
**Pull request by og76**
seams not to break prev

### CODE REVIEW

**Comment by og76** [comment]
should we also allow X,Y instead of AB for sockets?

File: nodes/number/float_math.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
     ("ARCSINE", "Arcsine", "asin A", "", 7),
     ("ARCCOSINE", "Arccosine", "acos A", "", 8),
     ("ARCTANGENT", "Arctangent", "atan A", "", 9),
+
```
Comment (JacquesLucke) [inline nodes/number/float_math.py]: was this new line intended?

**Comment by JacquesLucke** [comment]
I'm thinking... Yeah looks like, A and B make no sense for the atan2 operation..
So yes, we can use X, Y here.

File: nodes/number/float_math.py (hunk 1 above)
Comment (og76) [inline nodes/number/float_math.py]: actually, I had others after 3 and 18 to separate more visually (the list is soo long)
I can remove it if u think is useless, 
or add the others (to split in math/trigo/ etc)

**Pull request merged**